externs/browser/w3c_indexeddb.js

```diff
@@ -572,35 +471,52 @@ IDBIndex.prototype.keyPath;
  * @type {boolean}
  * @const
  */
+IDBIndex.prototype.multiEntry;
+
+/**
+ * @type {boolean}
+ * @const
+ */
 IDBIndex.prototype.unique;
 
 /**
- * @param {IDBKeyRange=} range The range of the cursor.
- * @param {(number|string)=} direction The direction of cursor enumeration.
+ * TODO(vobruba-martin): Change direction param to !IDBCursorDirection once CC understands string literal types.
+ * 
+ * @param {(!IDBKeyType|!IDBKeyRange)=} range The range of the cursor.
+ * @param {(!IDBCursorDirection|string)=} direction The direction of cursor enumeration.
```

Comment:
Sadly it isn't safe to disallow nulls for these arguments.
This definitely breaks some of our existing code that uses openCursor and openKeyCursor